'gb build' fails to build github.com/zeromq/goczmq on linux

I've just tried to compile goczmq on debian wheezy, this works with go build but with gb build, I get the following output:

```
luna@dev:~/src/gb/delta/vendor/src/github.com/zeromq/goczmq$ gb build
/tmp/gb344207216/github.com/zeromq/auth.cgo2.o: In function `Auth_new':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/auth.go:7: undefined reference to `zauth'
/tmp/gb344207216/github.com/zeromq/auth.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_wait':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/auth.go:63: undefined reference to `zsock_wait'
/tmp/gb344207216/github.com/zeromq/auth.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zstr_send':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/auth.go:78: undefined reference to `zstr_send'
/tmp/gb344207216/github.com/zeromq/auth.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zstr_sendm':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/auth.go:93: undefined reference to `zstr_sendm'
/tmp/gb344207216/github.com/zeromq/auth.cgo2.o: In function `Auth_new':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/auth.go:7: undefined reference to `zactor_new'
/tmp/gb344207216/github.com/zeromq/auth.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zactor_destroy':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/auth.go:51: undefined reference to `zactor_destroy'
/tmp/gb344207216/github.com/zeromq/beacon.cgo2.o: In function `Beacon_new':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/beacon.go:7: undefined reference to `zbeacon'
/tmp/gb344207216/github.com/zeromq/beacon.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zstr_recv':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/beacon.go:64: undefined reference to `zstr_recv'
/tmp/gb344207216/github.com/zeromq/beacon.cgo2.o: In function `Beacon_new':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/beacon.go:7: undefined reference to `zactor_new'
/tmp/gb344207216/github.com/zeromq/beacon.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_rcvtimeo':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/beacon.go:53: undefined reference to `zsock_set_rcvtimeo'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `Set_meta':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:6: undefined reference to `zcert_set_meta'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_dup':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:64: undefined reference to `zcert_dup'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_eq':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:79: undefined reference to `zcert_eq'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_load':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:92: undefined reference to `zcert_load'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_meta':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:106: undefined reference to `zcert_meta'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_new':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:118: undefined reference to `zcert_new'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_new_from':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:132: undefined reference to `zcert_new_from'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_public_key':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:154: undefined reference to `zcert_public_key'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_public_txt':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:167: undefined reference to `zcert_public_txt'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_save':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:182: undefined reference to `zcert_save'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_save_public':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:197: undefined reference to `zcert_save_public'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_save_secret':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:212: undefined reference to `zcert_save_secret'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_secret_key':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:225: undefined reference to `zcert_secret_key'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_resolve':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:238: undefined reference to `zsock_resolve'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_destroy':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:53: undefined reference to `zcert_destroy'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_print':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:143: undefined reference to `zcert_print'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsocket_set_curve_publickey_bin':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:250: undefined reference to `zsocket_set_curve_publickey_bin'
/tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsocket_set_curve_secretkey_bin':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:260: undefined reference to `zsocket_set_curve_secretkey_bin'
/tmp/gb344207216/github.com/zeromq/certstore.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcertstore_lookup':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/certstore.go:57: undefined reference to `zcertstore_lookup'
/tmp/gb344207216/github.com/zeromq/certstore.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcertstore_new':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/certstore.go:70: undefined reference to `zcertstore_new'
/tmp/gb344207216/github.com/zeromq/certstore.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcertstore_destroy':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/certstore.go:35: undefined reference to `zcertstore_destroy'
/tmp/gb344207216/github.com/zeromq/certstore.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcertstore_insert':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/certstore.go:45: undefined reference to `zcertstore_insert'
/tmp/gb344207216/github.com/zeromq/certstore.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcertstore_print':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/certstore.go:81: undefined reference to `zcertstore_print'
/tmp/gb344207216/github.com/zeromq/gossip.cgo2.o: In function `Gossip_new':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/gossip.go:7: undefined reference to `zgossip'
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/gossip.go:7: undefined reference to `zactor_new'
/tmp/gb344207216/github.com/zeromq/poller.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zpoller_add':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/poller.go:57: undefined reference to `zpoller_add'
/tmp/gb344207216/github.com/zeromq/poller.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zpoller_wait':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/poller.go:81: undefined reference to `zpoller_wait'
/tmp/gb344207216/github.com/zeromq/poller.cgo2.o: In function `Poller_new':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/poller.go:7: undefined reference to `zpoller_new'
/tmp/gb344207216/github.com/zeromq/poller.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zpoller_destroy':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/poller.go:68: undefined reference to `zpoller_destroy'
/tmp/gb344207216/github.com/zeromq/proxy.cgo2.o: In function `Zproxy_new':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/proxy.go:7: undefined reference to `zproxy'
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/proxy.go:7: undefined reference to `zactor_new'
/tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zframe_data':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:104: undefined reference to `zframe_data'
/tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zframe_more':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:127: undefined reference to `zframe_more'
/tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zframe_new':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:141: undefined reference to `zframe_new'
/tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zframe_recv':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:154: undefined reference to `zframe_recv'
/tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zframe_send':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:171: undefined reference to `zframe_send'
/tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zframe_size':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:184: undefined reference to `zframe_size'
/tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_attach':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:201: undefined reference to `zsock_attach'
/tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_new_checked':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:228: undefined reference to `zsock_new_checked'
/tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `Sock_connect':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:8: undefined reference to `zsock_connect'
/tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `Sock_disconnect':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:9: undefined reference to `zsock_disconnect'
/tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `Sock_bind':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:10: undefined reference to `zsock_bind'
/tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `Sock_unbind':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:11: undefined reference to `zsock_unbind'
/tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zframe_destroy':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:115: undefined reference to `zframe_destroy'
/tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_destroy_checked':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:214: undefined reference to `zsock_destroy_checked'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_affinity':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:60: undefined reference to `zsock_affinity'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_backlog':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:74: undefined reference to `zsock_backlog'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_curve_publickey':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:87: undefined reference to `zsock_curve_publickey'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_curve_secretkey':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:100: undefined reference to `zsock_curve_secretkey'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_curve_server':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:114: undefined reference to `zsock_curve_server'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_curve_serverkey':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:127: undefined reference to `zsock_curve_serverkey'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_events':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:141: undefined reference to `zsock_events'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_fd':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:155: undefined reference to `zsock_fd'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_gssapi_plaintext':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:169: undefined reference to `zsock_gssapi_plaintext'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_gssapi_principal':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:182: undefined reference to `zsock_gssapi_principal'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_gssapi_server':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:196: undefined reference to `zsock_gssapi_server'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_gssapi_service_principal':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:209: undefined reference to `zsock_gssapi_service_principal'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_identity':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:222: undefined reference to `zsock_identity'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_immediate':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:236: undefined reference to `zsock_immediate'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_ipv4only':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:250: undefined reference to `zsock_ipv4only'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_ipv6':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:264: undefined reference to `zsock_ipv6'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_last_endpoint':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:277: undefined reference to `zsock_last_endpoint'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_linger':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:291: undefined reference to `zsock_linger'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_maxmsgsize':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:305: undefined reference to `zsock_maxmsgsize'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_mechanism':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:319: undefined reference to `zsock_mechanism'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_multicast_hops':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:333: undefined reference to `zsock_multicast_hops'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_plain_password':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:346: undefined reference to `zsock_plain_password'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_plain_server':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:360: undefined reference to `zsock_plain_server'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_plain_username':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:373: undefined reference to `zsock_plain_username'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_rate':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:387: undefined reference to `zsock_rate'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_rcvbuf':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:401: undefined reference to `zsock_rcvbuf'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_rcvhwm':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:415: undefined reference to `zsock_rcvhwm'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_rcvmore':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:429: undefined reference to `zsock_rcvmore'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_rcvtimeo':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:443: undefined reference to `zsock_rcvtimeo'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_reconnect_ivl':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:457: undefined reference to `zsock_reconnect_ivl'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_reconnect_ivl_max':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:471: undefined reference to `zsock_reconnect_ivl_max'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_recovery_ivl':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:485: undefined reference to `zsock_recovery_ivl'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_sndbuf':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1004: undefined reference to `zsock_sndbuf'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_sndhwm':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1018: undefined reference to `zsock_sndhwm'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_sndtimeo':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1032: undefined reference to `zsock_sndtimeo'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_tcp_accept_filter':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1045: undefined reference to `zsock_tcp_accept_filter'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_tcp_keepalive':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1059: undefined reference to `zsock_tcp_keepalive'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_tcp_keepalive_cnt':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1073: undefined reference to `zsock_tcp_keepalive_cnt'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_tcp_keepalive_idle':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1087: undefined reference to `zsock_tcp_keepalive_idle'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_tcp_keepalive_intvl':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1101: undefined reference to `zsock_tcp_keepalive_intvl'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_tos':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1115: undefined reference to `zsock_tos'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_type':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1129: undefined reference to `zsock_type'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_zap_domain':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1142: undefined reference to `zsock_zap_domain'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_affinity':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:498: undefined reference to `zsock_set_affinity'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_backlog':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:509: undefined reference to `zsock_set_backlog'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_conflate':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:520: undefined reference to `zsock_set_conflate'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_curve_publickey':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:530: undefined reference to `zsock_set_curve_publickey'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_curve_secretkey':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:540: undefined reference to `zsock_set_curve_secretkey'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_curve_server':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:551: undefined reference to `zsock_set_curve_server'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_curve_serverkey':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:561: undefined reference to `zsock_set_curve_serverkey'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_delay_attach_on_connect':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:572: undefined reference to `zsock_set_delay_attach_on_connect'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_gssapi_plaintext':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:583: undefined reference to `zsock_set_gssapi_plaintext'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_gssapi_principal':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:593: undefined reference to `zsock_set_gssapi_principal'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_gssapi_server':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:604: undefined reference to `zsock_set_gssapi_server'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_gssapi_service_principal':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:614: undefined reference to `zsock_set_gssapi_service_principal'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_identity':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:624: undefined reference to `zsock_set_identity'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_immediate':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:635: undefined reference to `zsock_set_immediate'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_ipv4only':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:646: undefined reference to `zsock_set_ipv4only'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_ipv6':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:657: undefined reference to `zsock_set_ipv6'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_linger':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:668: undefined reference to `zsock_set_linger'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_maxmsgsize':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:679: undefined reference to `zsock_set_maxmsgsize'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_multicast_hops':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:690: undefined reference to `zsock_set_multicast_hops'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_plain_password':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:700: undefined reference to `zsock_set_plain_password'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_plain_server':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:711: undefined reference to `zsock_set_plain_server'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_plain_username':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:721: undefined reference to `zsock_set_plain_username'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_probe_router':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:732: undefined reference to `zsock_set_probe_router'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_rate':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:743: undefined reference to `zsock_set_rate'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_rcvbuf':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:754: undefined reference to `zsock_set_rcvbuf'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_rcvhwm':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:765: undefined reference to `zsock_set_rcvhwm'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_reconnect_ivl':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:776: undefined reference to `zsock_set_reconnect_ivl'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_reconnect_ivl_max':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:787: undefined reference to `zsock_set_reconnect_ivl_max'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_recovery_ivl':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:798: undefined reference to `zsock_set_recovery_ivl'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_req_correlate':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:809: undefined reference to `zsock_set_req_correlate'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_req_relaxed':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:820: undefined reference to `zsock_set_req_relaxed'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_router_handover':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:831: undefined reference to `zsock_set_router_handover'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_router_mandatory':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:842: undefined reference to `zsock_set_router_mandatory'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_router_raw':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:853: undefined reference to `zsock_set_router_raw'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_sndbuf':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:864: undefined reference to `zsock_set_sndbuf'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_sndhwm':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:875: undefined reference to `zsock_set_sndhwm'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_sndtimeo':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:886: undefined reference to `zsock_set_sndtimeo'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_subscribe':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:896: undefined reference to `zsock_set_subscribe'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_tcp_accept_filter':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:906: undefined reference to `zsock_set_tcp_accept_filter'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_tcp_keepalive':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:917: undefined reference to `zsock_set_tcp_keepalive'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_tcp_keepalive_cnt':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:928: undefined reference to `zsock_set_tcp_keepalive_cnt'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_tcp_keepalive_idle':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:939: undefined reference to `zsock_set_tcp_keepalive_idle'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_tcp_keepalive_intvl':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:950: undefined reference to `zsock_set_tcp_keepalive_intvl'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_tos':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:961: undefined reference to `zsock_set_tos'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_unsubscribe':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:971: undefined reference to `zsock_set_unsubscribe'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_xpub_verbose':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:982: undefined reference to `zsock_set_xpub_verbose'
/tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_zap_domain':
/home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:992: undefined reference to `zsock_set_zap_domain'
collect2: error: ld returned 1 exit status
FATAL command "build" failed: # gcc -fPIC -m64 -pthread -fmessage-length=0 -o /tmp/gb344207216/github.com/zeromq/_cgo_.o /tmp/gb344207216/github.com/zeromq/_cgo_main.o /tmp/gb344207216/github.com/zeromq/_cgo_export.o /tmp/gb344207216/github.com/zeromq/auth.cgo2.o /tmp/gb344207216/github.com/zeromq/beacon.cgo2.o /tmp/gb344207216/github.com/zeromq/cert.cgo2.o /tmp/gb344207216/github.com/zeromq/certstore.cgo2.o /tmp/gb344207216/github.com/zeromq/goczmq.cgo2.o /tmp/gb344207216/github.com/zeromq/gossip.cgo2.o /tmp/gb344207216/github.com/zeromq/poller.cgo2.o /tmp/gb344207216/github.com/zeromq/proxy.cgo2.o /tmp/gb344207216/github.com/zeromq/sock.cgo2.o /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o -g -O2: exit status 1

```

Thanks for the error report, I suspect this is missing pkg-config support



> On 24 May 2015, at 20:02, Luna Duclos <notifications@github.com> wrote:
> 
> I've just tried to compile goczmq on debian wheezy, this works with go build but with gb build, I get the following output:
> 
> luna@dev:~/src/gb/delta/vendor/src/github.com/zeromq/goczmq$ gb build
> /tmp/gb344207216/github.com/zeromq/auth.cgo2.o: In function `Auth_new':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/auth.go:7: undefined reference to `zauth'
> /tmp/gb344207216/github.com/zeromq/auth.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_wait':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/auth.go:63: undefined reference to `zsock_wait'
> /tmp/gb344207216/github.com/zeromq/auth.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zstr_send':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/auth.go:78: undefined reference to `zstr_send'
> /tmp/gb344207216/github.com/zeromq/auth.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zstr_sendm':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/auth.go:93: undefined reference to `zstr_sendm'
> /tmp/gb344207216/github.com/zeromq/auth.cgo2.o: In function `Auth_new':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/auth.go:7: undefined reference to `zactor_new'
> /tmp/gb344207216/github.com/zeromq/auth.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zactor_destroy':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/auth.go:51: undefined reference to `zactor_destroy'
> /tmp/gb344207216/github.com/zeromq/beacon.cgo2.o: In function `Beacon_new':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/beacon.go:7: undefined reference to `zbeacon'
> /tmp/gb344207216/github.com/zeromq/beacon.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zstr_recv':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/beacon.go:64: undefined reference to `zstr_recv'
> /tmp/gb344207216/github.com/zeromq/beacon.cgo2.o: In function `Beacon_new':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/beacon.go:7: undefined reference to `zactor_new'
> /tmp/gb344207216/github.com/zeromq/beacon.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_rcvtimeo':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/beacon.go:53: undefined reference to `zsock_set_rcvtimeo'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `Set_meta':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:6: undefined reference to `zcert_set_meta'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_dup':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:64: undefined reference to `zcert_dup'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_eq':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:79: undefined reference to `zcert_eq'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_load':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:92: undefined reference to `zcert_load'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_meta':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:106: undefined reference to `zcert_meta'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_new':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:118: undefined reference to `zcert_new'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_new_from':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:132: undefined reference to `zcert_new_from'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_public_key':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:154: undefined reference to `zcert_public_key'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_public_txt':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:167: undefined reference to `zcert_public_txt'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_save':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:182: undefined reference to `zcert_save'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_save_public':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:197: undefined reference to `zcert_save_public'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_save_secret':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:212: undefined reference to `zcert_save_secret'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_secret_key':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:225: undefined reference to `zcert_secret_key'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_resolve':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:238: undefined reference to `zsock_resolve'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_destroy':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:53: undefined reference to `zcert_destroy'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcert_print':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:143: undefined reference to `zcert_print'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsocket_set_curve_publickey_bin':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:250: undefined reference to `zsocket_set_curve_publickey_bin'
> /tmp/gb344207216/github.com/zeromq/cert.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsocket_set_curve_secretkey_bin':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/cert.go:260: undefined reference to `zsocket_set_curve_secretkey_bin'
> /tmp/gb344207216/github.com/zeromq/certstore.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcertstore_lookup':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/certstore.go:57: undefined reference to `zcertstore_lookup'
> /tmp/gb344207216/github.com/zeromq/certstore.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcertstore_new':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/certstore.go:70: undefined reference to `zcertstore_new'
> /tmp/gb344207216/github.com/zeromq/certstore.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcertstore_destroy':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/certstore.go:35: undefined reference to `zcertstore_destroy'
> /tmp/gb344207216/github.com/zeromq/certstore.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcertstore_insert':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/certstore.go:45: undefined reference to `zcertstore_insert'
> /tmp/gb344207216/github.com/zeromq/certstore.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zcertstore_print':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/certstore.go:81: undefined reference to `zcertstore_print'
> /tmp/gb344207216/github.com/zeromq/gossip.cgo2.o: In function `Gossip_new':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/gossip.go:7: undefined reference to `zgossip'
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/gossip.go:7: undefined reference to `zactor_new'
> /tmp/gb344207216/github.com/zeromq/poller.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zpoller_add':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/poller.go:57: undefined reference to `zpoller_add'
> /tmp/gb344207216/github.com/zeromq/poller.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zpoller_wait':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/poller.go:81: undefined reference to `zpoller_wait'
> /tmp/gb344207216/github.com/zeromq/poller.cgo2.o: In function `Poller_new':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/poller.go:7: undefined reference to `zpoller_new'
> /tmp/gb344207216/github.com/zeromq/poller.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zpoller_destroy':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/poller.go:68: undefined reference to `zpoller_destroy'
> /tmp/gb344207216/github.com/zeromq/proxy.cgo2.o: In function `Zproxy_new':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/proxy.go:7: undefined reference to `zproxy'
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/proxy.go:7: undefined reference to `zactor_new'
> /tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zframe_data':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:104: undefined reference to `zframe_data'
> /tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zframe_more':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:127: undefined reference to `zframe_more'
> /tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zframe_new':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:141: undefined reference to `zframe_new'
> /tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zframe_recv':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:154: undefined reference to `zframe_recv'
> /tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zframe_send':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:171: undefined reference to `zframe_send'
> /tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zframe_size':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:184: undefined reference to `zframe_size'
> /tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_attach':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:201: undefined reference to `zsock_attach'
> /tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_new_checked':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:228: undefined reference to `zsock_new_checked'
> /tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `Sock_connect':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:8: undefined reference to `zsock_connect'
> /tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `Sock_disconnect':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:9: undefined reference to `zsock_disconnect'
> /tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `Sock_bind':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:10: undefined reference to `zsock_bind'
> /tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `Sock_unbind':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:11: undefined reference to `zsock_unbind'
> /tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zframe_destroy':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:115: undefined reference to `zframe_destroy'
> /tmp/gb344207216/github.com/zeromq/sock.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_destroy_checked':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock.go:214: undefined reference to `zsock_destroy_checked'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_affinity':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:60: undefined reference to `zsock_affinity'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_backlog':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:74: undefined reference to `zsock_backlog'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_curve_publickey':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:87: undefined reference to `zsock_curve_publickey'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_curve_secretkey':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:100: undefined reference to `zsock_curve_secretkey'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_curve_server':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:114: undefined reference to `zsock_curve_server'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_curve_serverkey':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:127: undefined reference to `zsock_curve_serverkey'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_events':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:141: undefined reference to `zsock_events'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_fd':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:155: undefined reference to `zsock_fd'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_gssapi_plaintext':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:169: undefined reference to `zsock_gssapi_plaintext'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_gssapi_principal':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:182: undefined reference to `zsock_gssapi_principal'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_gssapi_server':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:196: undefined reference to `zsock_gssapi_server'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_gssapi_service_principal':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:209: undefined reference to `zsock_gssapi_service_principal'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_identity':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:222: undefined reference to `zsock_identity'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_immediate':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:236: undefined reference to `zsock_immediate'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_ipv4only':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:250: undefined reference to `zsock_ipv4only'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_ipv6':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:264: undefined reference to `zsock_ipv6'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_last_endpoint':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:277: undefined reference to `zsock_last_endpoint'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_linger':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:291: undefined reference to `zsock_linger'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_maxmsgsize':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:305: undefined reference to `zsock_maxmsgsize'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_mechanism':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:319: undefined reference to `zsock_mechanism'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_multicast_hops':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:333: undefined reference to `zsock_multicast_hops'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_plain_password':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:346: undefined reference to `zsock_plain_password'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_plain_server':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:360: undefined reference to `zsock_plain_server'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_plain_username':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:373: undefined reference to `zsock_plain_username'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_rate':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:387: undefined reference to `zsock_rate'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_rcvbuf':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:401: undefined reference to `zsock_rcvbuf'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_rcvhwm':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:415: undefined reference to `zsock_rcvhwm'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_rcvmore':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:429: undefined reference to `zsock_rcvmore'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_rcvtimeo':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:443: undefined reference to `zsock_rcvtimeo'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_reconnect_ivl':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:457: undefined reference to `zsock_reconnect_ivl'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_reconnect_ivl_max':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:471: undefined reference to `zsock_reconnect_ivl_max'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_recovery_ivl':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:485: undefined reference to `zsock_recovery_ivl'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_sndbuf':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1004: undefined reference to `zsock_sndbuf'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_sndhwm':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1018: undefined reference to `zsock_sndhwm'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_sndtimeo':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1032: undefined reference to `zsock_sndtimeo'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_tcp_accept_filter':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1045: undefined reference to `zsock_tcp_accept_filter'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_tcp_keepalive':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1059: undefined reference to `zsock_tcp_keepalive'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_tcp_keepalive_cnt':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1073: undefined reference to `zsock_tcp_keepalive_cnt'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_tcp_keepalive_idle':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1087: undefined reference to `zsock_tcp_keepalive_idle'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_tcp_keepalive_intvl':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1101: undefined reference to `zsock_tcp_keepalive_intvl'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_tos':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1115: undefined reference to `zsock_tos'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_type':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1129: undefined reference to `zsock_type'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_zap_domain':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:1142: undefined reference to `zsock_zap_domain'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_affinity':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:498: undefined reference to `zsock_set_affinity'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_backlog':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:509: undefined reference to `zsock_set_backlog'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_conflate':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:520: undefined reference to `zsock_set_conflate'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_curve_publickey':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:530: undefined reference to `zsock_set_curve_publickey'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_curve_secretkey':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:540: undefined reference to `zsock_set_curve_secretkey'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_curve_server':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:551: undefined reference to `zsock_set_curve_server'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_curve_serverkey':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:561: undefined reference to `zsock_set_curve_serverkey'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_delay_attach_on_connect':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:572: undefined reference to `zsock_set_delay_attach_on_connect'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_gssapi_plaintext':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:583: undefined reference to `zsock_set_gssapi_plaintext'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_gssapi_principal':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:593: undefined reference to `zsock_set_gssapi_principal'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_gssapi_server':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:604: undefined reference to `zsock_set_gssapi_server'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_gssapi_service_principal':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:614: undefined reference to `zsock_set_gssapi_service_principal'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_identity':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:624: undefined reference to `zsock_set_identity'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_immediate':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:635: undefined reference to `zsock_set_immediate'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_ipv4only':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:646: undefined reference to `zsock_set_ipv4only'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_ipv6':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:657: undefined reference to `zsock_set_ipv6'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_linger':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:668: undefined reference to `zsock_set_linger'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_maxmsgsize':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:679: undefined reference to `zsock_set_maxmsgsize'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_multicast_hops':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:690: undefined reference to `zsock_set_multicast_hops'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_plain_password':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:700: undefined reference to `zsock_set_plain_password'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_plain_server':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:711: undefined reference to `zsock_set_plain_server'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_plain_username':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:721: undefined reference to `zsock_set_plain_username'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_probe_router':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:732: undefined reference to `zsock_set_probe_router'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_rate':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:743: undefined reference to `zsock_set_rate'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_rcvbuf':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:754: undefined reference to `zsock_set_rcvbuf'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_rcvhwm':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:765: undefined reference to `zsock_set_rcvhwm'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_reconnect_ivl':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:776: undefined reference to `zsock_set_reconnect_ivl'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_reconnect_ivl_max':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:787: undefined reference to `zsock_set_reconnect_ivl_max'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_recovery_ivl':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:798: undefined reference to `zsock_set_recovery_ivl'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_req_correlate':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:809: undefined reference to `zsock_set_req_correlate'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_req_relaxed':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:820: undefined reference to `zsock_set_req_relaxed'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_router_handover':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:831: undefined reference to `zsock_set_router_handover'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_router_mandatory':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:842: undefined reference to `zsock_set_router_mandatory'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_router_raw':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:853: undefined reference to `zsock_set_router_raw'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_sndbuf':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:864: undefined reference to `zsock_set_sndbuf'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_sndhwm':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:875: undefined reference to `zsock_set_sndhwm'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_sndtimeo':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:886: undefined reference to `zsock_set_sndtimeo'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_subscribe':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:896: undefined reference to `zsock_set_subscribe'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_tcp_accept_filter':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:906: undefined reference to `zsock_set_tcp_accept_filter'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_tcp_keepalive':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:917: undefined reference to `zsock_set_tcp_keepalive'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_tcp_keepalive_cnt':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:928: undefined reference to `zsock_set_tcp_keepalive_cnt'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_tcp_keepalive_idle':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:939: undefined reference to `zsock_set_tcp_keepalive_idle'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_tcp_keepalive_intvl':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:950: undefined reference to `zsock_set_tcp_keepalive_intvl'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_tos':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:961: undefined reference to `zsock_set_tos'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_unsubscribe':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:971: undefined reference to `zsock_set_unsubscribe'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_xpub_verbose':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:982: undefined reference to `zsock_set_xpub_verbose'
> /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o: In function `_cgo_ccd0f2292f0d_Cfunc_zsock_set_zap_domain':
> /home/luna/src/gb/delta/vendor/src/github.com/zeromq/goczmq/sock_option.go:992: undefined reference to `zsock_set_zap_domain'
> collect2: error: ld returned 1 exit status
> FATAL command "build" failed: # gcc -fPIC -m64 -pthread -fmessage-length=0 -o /tmp/gb344207216/github.com/zeromq/_cgo_.o /tmp/gb344207216/github.com/zeromq/_cgo_main.o /tmp/gb344207216/github.com/zeromq/_cgo_export.o /tmp/gb344207216/github.com/zeromq/auth.cgo2.o /tmp/gb344207216/github.com/zeromq/beacon.cgo2.o /tmp/gb344207216/github.com/zeromq/cert.cgo2.o /tmp/gb344207216/github.com/zeromq/certstore.cgo2.o /tmp/gb344207216/github.com/zeromq/goczmq.cgo2.o /tmp/gb344207216/github.com/zeromq/gossip.cgo2.o /tmp/gb344207216/github.com/zeromq/poller.cgo2.o /tmp/gb344207216/github.com/zeromq/proxy.cgo2.o /tmp/gb344207216/github.com/zeromq/sock.cgo2.o /tmp/gb344207216/github.com/zeromq/sock_option.cgo2.o -g -O2: exit status 1
> 
> —
> Reply to this email directly or view it on GitHub.
>

--- a/cgo.go
+++ b/cgo.go
@@ -35,7 +35,7 @@ func rungcc1(pkg *Package, ofile, cfile string) Target {
 }
 
 // rungcc2 links the o files from rungcc1 into a single _cgo_.o.
-func rungcc2(pkg *Package, ofile string, ofiles []string) error {
+func rungcc2(pkg *Package, cgoCFLAGS, cgoLDFLAGS []string, ofile string, ofiles []string) error {
 	args := []string{
 		"-fPIC", "-m64", "-fmessage-length=0",
 	}
@@ -44,7 +44,6 @@ func rungcc2(pkg *Package, ofile string, ofiles []string) error {
 	}
 	args = append(args, "-o", ofile)
 	args = append(args, ofiles...)
-	_, _, _, cgoLDFLAGS := cflags(pkg, true)
 	args = append(args, cgoLDFLAGS...) // this has to go at the end, because reasons!
 	t0 := time.Now()
 	err := pkg.run(pkg.Dir, nil, gcc(), args...)
@@ -183,4 +182,12 @@ func pkgconfig(p *Package) ([]string, []string, error) {
 	}
 	ldflags := strings.Fields(out.String())
 	return cflags, ldflags, nil
+}
+
+func quoteFlags(flags []string) []string {
+	quoted := make([]string, len(flags))
+	for i, f := range flags {
+		quoted[i] = strconv.Quote(f)
+	}
+	return quoted
 }

--- a/cgo.go
+++ b/cgo.go
@@ -5,6 +5,7 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
+	"strconv"
 	"strings"
 	"time"
 )

--- a/cgo14.go
+++ b/cgo14.go
@@ -17,14 +17,15 @@ func cgo(pkg *Package) ([]ObjTarget, []string) {
 	fn := func(t ...ObjTarget) ([]ObjTarget, []string) {
 		return t, nil
 	}
-	_, _, _, cgoLDFLAGS := cflags(pkg, true)
-	_, pcLDFLAGS, err := pkgconfig(pkg)
+	_, _, cgoCFLAGS, cgoLDFLAGS := cflags(pkg, true)
+	pcCFLAGS, pcLDFLAGS, err := pkgconfig(pkg)
 	if err != nil {
 		return fn(ErrTarget{err})
 	}
+	cgoCFLAGS = append(cgoCFLAGS, pcCFLAGS...)
 	cgoLDFLAGS = append(cgoLDFLAGS, pcLDFLAGS...)
 
-	if err := runcgo1(pkg, nil, cgoLDFLAGS); err != nil {
+	if err := runcgo1(pkg, cgoCFLAGS, cgoLDFLAGS); err != nil {
 		return fn(ErrTarget{err})
 	}
 
@@ -57,7 +58,7 @@ func cgo(pkg *Package) ([]ObjTarget, []string) {
 	}
 
 	ofile := filepath.Join(filepath.Dir(ofiles[0]), "_cgo_.o")
-	if err := rungcc2(pkg, ofile, ofiles); err != nil {
+	if err := rungcc2(pkg, cgoCFLAGS, cgoLDFLAGS, ofile, ofiles); err != nil {
 		return fn(ErrTarget{err})
 	}
 
@@ -93,14 +94,9 @@ func runcgo1(pkg *Package, cflags, ldflags []string) error {
 	}
 	args = append(args, pkg.CgoFiles...)
 
-	// Update $CGO_LDFLAGS with p.CgoLDFLAGS.
-	var cgoenv []string
-	if len(ldflags) > 0 {
-		flags := make([]string, len(ldflags))
-		for i, f := range ldflags {
-			flags[i] = strconv.Quote(f)
-		}
-		cgoenv = []string{"CGO_LDFLAGS=" + strings.Join(flags, " ")}
+	cgoenv := []string{
+		"CGO_CFLAGS=" + strings.Join(quoteFlags(cflags), " "),
+		"CGO_LDFLAGS=" + strings.Join(quoteFlags(ldflags), " "),
 	}
 	return pkg.run(pkg.Dir, cgoenv, cgo, args...)
 }

--- a/cgo14.go
+++ b/cgo14.go
@@ -4,7 +4,6 @@ package gb
 
 import (
 	"path/filepath"
-	"strconv"
 	"strings"
 )
 

--- a/cgo15.go
+++ b/cgo15.go
@@ -4,7 +4,6 @@ package gb
 
 import (
 	"path/filepath"
-	"strconv"
 	"strings"
 )
 

--- a/cgo15.go
+++ b/cgo15.go
@@ -17,13 +17,14 @@ func cgo(pkg *Package) ([]ObjTarget, []string) {
 	fn := func(t ...ObjTarget) ([]ObjTarget, []string) {
 		return t, nil
 	}
-	_, _, _, cgoLDFLAGS := cflags(pkg, true)
-	_, pcLDFLAGS, err := pkgconfig(pkg)
+	_, _, cgoCFLAGS, cgoLDFLAGS := cflags(pkg, true)
+	pcCFLAGS, pcLDFLAGS, err := pkgconfig(pkg)
 	if err != nil {
 		return fn(ErrTarget{err})
 	}
+	cgoCFLAGS = append(cgoCFLAGS, pcCFLAGS...)
 	cgoLDFLAGS = append(cgoLDFLAGS, pcLDFLAGS...)
-	if err := runcgo1(pkg, nil, cgoLDFLAGS); err != nil {
+	if err := runcgo1(pkg, cgoCFLAGS, cgoLDFLAGS); err != nil {
 		return fn(ErrTarget{err})
 	}
 
@@ -56,7 +57,7 @@ func cgo(pkg *Package) ([]ObjTarget, []string) {
 	}
 
 	ofile := filepath.Join(filepath.Dir(ofiles[0]), "_cgo_.o")
-	if err := rungcc2(pkg, ofile, ofiles); err != nil {
+	if err := rungcc2(pkg, cgoCFLAGS, cgoLDFLAGS, ofile, ofiles); err != nil {
 		return fn(ErrTarget{err})
 	}
 
@@ -92,13 +93,9 @@ func runcgo1(pkg *Package, cflags, ldflags []string) error {
 	args = append(args, pkg.CgoFiles...)
 
 	// Update $CGO_LDFLAGS with p.CgoLDFLAGS.
-	var cgoenv []string
-	if len(ldflags) > 0 {
-		flags := make([]string, len(ldflags))
-		for i, f := range ldflags {
-			flags[i] = strconv.Quote(f)
-		}
-		cgoenv = []string{"CGO_LDFLAGS=" + strings.Join(flags, " ")}
+	cgoenv := []string{
+		"CGO_CFLAGS=" + strings.Join(quoteFlags(cflags), " "),
+		"CGO_LDFLAGS=" + strings.Join(quoteFlags(ldflags), " "),
 	}
 	return pkg.run(pkg.Dir, cgoenv, cgo, args...)
 }

--- a/wercker.yml
+++ b/wercker.yml
@@ -22,9 +22,21 @@ build:
 
     # Integration tests
     - script:
+        name: integation test/setup
+        code:
+          git clone --quiet https://github.com/constabulary/integration-tests 
+
+    - script:
         name: integration test/sqlite3
         code: | 
-          cd ${WERCKER_STEP_TEMP}
-          git clone --quiet https://github.com/constabulary/integration-tests 
-          gb vendor -v -R integration-tests/sqlite3 restore
-          gb build -v -R integration-tests/sqlite3
+          gb vendor -R integration-tests/sqlite3 restore
+          gb build -R integration-tests/sqlite3
+
+    - script:
+        name: intergration test/goczmq
+        code: |
+          cd integration-tests/goczmq
+          bash -x setup.bash
+          gb vendor -v restore
+          gb build
+	  

